_schemaV2.graphql

```diff
@@ -22380,27 +22333,23 @@ input ViewingRoomSubsectionInput {
   internalID: ID
 }
 
-# The connection type for ViewingRoom.
+# A connection to a list of items.
 type ViewingRoomsConnection {
   # A list of edges.
   edges: [ViewingRoomsEdge]
-
-  # A list of nodes.
-  nodes: [ViewingRoom]
```

Comment:
argh, really don't want to add noted right to ViewingRoomsConnection (without edges in the middle). Is there a guaranteed way to tell wether there are any clients which do viewingRoomsConnection.nodes?